src/Nncase.Tests/Quant/config_leaky.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
There may be other jsons in future. Naming the file `*.quant.json` to help users classifing jsons.